css-structure/css-structure_selectors-and-cascading/css/styles.css

```diff
@@ -49,92 +49,94 @@ nav li {
 /* -------------------------------------------------------- */
 
 /* 1. These styles should apply to the headline */
-FIX_ME {
+h1 {
   outline: 2px dashed red;
```

Comment:
I would comment out this line in all code

Reply:
done